playbooks/common/openshift-node/config.yml

```diff
@@ -139,6 +139,8 @@
 - name: Set scheduleability
   hosts: oo_first_master
   vars:
+    openshift_nodes: "{{ hostvars
+                         | oo_select_keys(groups['oo_nodes_to_config']) }}"
```

Comment:
per @dlbewley's comment about openshift_node being a list of facts, this should actually be:
```
    openshift_nodes: "{{ hostvars
                         | oo_select_keys(groups['oo_nodes_to_config'])
                         | oo_collect('openshift.common.hostname') }}"
```